### PR TITLE
LPS-158442 Add basic mvc render commands and relax template path requirement

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:reevaluate:1327401_4811

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/configuration/icon/KBArticleSubscriptionPortletConfigurationIcon.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 	immediate = true,
 	property = {
 		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_ADMIN,
-		"path=/admin/view_article.jsp", "path=/admin/view_articles.jsp"
+		"path=/admin/view_articles.jsp", "path=/knowledge_base/view_article"
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/configuration/icon/KBArticleSubscriptionPortletConfigurationIcon.java:45]: When I'm trying to susbcribe > 

```
 ERROR [http-nio-8080-exec-3][PortletServlet:117] null
javax.portlet.PortletException: javax.portlet.PortletException: processAction method not implemented
	at com.liferay.portal.kernel.portlet.LiferayPortlet.callActionMethod(LiferayPortlet.java:192) ~[portal-kernel.jar:?]
	at com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet.callActionMethod(MVCPortlet.java:439) ~[portal-kernel.jar:?]
	at com.liferay.portal.kernel.portlet.LiferayPortlet.processAction(LiferayPortlet.java:82) ~[portal-kernel.jar:?]
	at com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet.processAction(MVCPortlet.java:256) ~[portal-kernel.jar:?]
	at com.liferay.portlet.internal.FilterChainImpl.doFilter(FilterChainImpl.java:77) ~[portal-impl.jar:?]
	at com.liferay.portal.kernel.portlet.PortletFilterUtil.doFilter(PortletFilterUtil.java:50) ~[portal-kernel.jar:?]
	at com.liferay.portal.kernel.servlet.PortletServlet.service(PortletServlet.java:113) ~[portal-kernel.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) ~[servlet-api.jar:4.0.FR]
	at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:153) ~[?:?]
	at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:62) ~[?:?]
	at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:120) ~[?:?]
	at org.eclipse.equinox.http.servlet.internal.servlet.RequestDispatcherAdaptor.include(RequestDispatcherAdaptor.java:48) ~[?:?]
	at com.liferay.portlet.internal.InvokerPortletImpl.invoke(InvokerPortletImpl.java:568) ~[portal-impl.jar:?]
	at com.liferay.portlet.internal.InvokerPortletImpl.invokeAction(InvokerPortletImpl.java:608) ~[portal-impl.jar:?]
	at com.liferay.portlet.internal.InvokerPortletImpl.processAction(InvokerPortletImpl.java:285) ~[portal-impl.jar:?]
	at com.liferay.portal.monitoring.internal.portlet.MonitoringInvokerPortlet.processAction(MonitoringInvokerPortlet.java:195) ~[?:?]
	at com.liferay.portlet.internal.PortletContainerImpl._processAction(PortletContainerImpl.java:514) ~[portal-impl.jar:?]
...	
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:9.0.56]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:9.0.56]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_311]
Caused by: javax.portlet.PortletException: processAction method not implemented
	at javax.portlet.GenericPortlet.processAction(GenericPortlet.java:223) ~[portlet.jar:3.0.1]
	at com.liferay.portal.kernel.portlet.LiferayPortlet.callActionMethod(LiferayPortlet.java:187) ~[portal-kernel.jar:?]
	... 166 more

```
Can you take a look?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
ci:test:knowledge-base

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:knowledge-base

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:forward

**Comment by austinchiang** [comment]
Test failures unrelated https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1914182155

**Comment by austinchiang** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/120787

**Pull request closed without merging**